components/eamxx/src/physics/nudging/atmosphere_nudging.cpp

```diff
@@ -0,0 +1,264 @@
+#include "atmosphere_nudging.hpp"
+
+namespace scream
+{
+
+  //using namespace spa;
+// =========================================================================================
+NUDGING::NUDGING (const ekat::Comm& comm, const ekat::ParameterList& params)
+  : AtmosphereProcess(comm, params)
+{
+  m_fnames=m_params.get<std::vector<std::string>>("Field_Names");
```

Comment:
Are the field names really something that the user can choose? It seems that the impl assumes we nudge T, qv, and horiz_winds. If that's always the case, I would not expose this entry in the input param list.

Reply:
I will remove it, this was originally there because we wanted to make it generic to fields that a user can choose. However, I later changed to just doing the fields we care about for the nudging.

Reply:
Removed